user-commands.go

```diff
@@ -434,11 +436,12 @@ func (adm *AdminClient) ListServiceAccounts(ctx context.Context, user string) (L
 
 // InfoServiceAccountResp is the response body of the info service account call
 type InfoServiceAccountResp struct {
-	ParentUser    string `json:"parentUser"`
-	AccountStatus string `json:"accountStatus"`
-	ImpliedPolicy bool   `json:"impliedPolicy"`
-	Policy        string `json:"policy"`
-	Comment       string `json:"comment"`
+	ParentUser    string    `json:"parentUser"`
+	AccountStatus string    `json:"accountStatus"`
+	ImpliedPolicy bool      `json:"impliedPolicy"`
+	Policy        string    `json:"policy"`
+	Comment       string    `json:"comment"`
+	Expiration    time.Time `json:"expiration,omitempty"`
```

Comment:
So we don't have to deal with zero values.
```suggestion
	Expiration    *time.Time `json:"expiration,omitempty"`
```